tools/tuner/protos/tuner.proto

```
@@ -0,0 +1,44 @@
+syntax = "proto3";
+
+package tuner;
```

Buf (1.58.0)

Files with package "tuner" must be within a directory "tuner" relative to root but were in directory "tools/tuner/protos".

(PACKAGE_DIRECTORY_MATCH)